drivers/SmartThings/zigbee-switch/fingerprints.yml

```diff
@@ -116,6 +116,26 @@ zigbeeManufacturer:
     manufacturer: Aqara
     model: lumi.light.acn014
     deviceProfileName: aqara-led-bulb
+  - id: "LUMI/lumi.switch.n1aeu1"
+    deviceLabel: Aqara Smart Wall Switch H1 EU (With Neutral, Single Rocker)
+    manufacturer: LUMI
+    model: lumi.switch.n1aeu1
+    deviceProfileName: aqara-switch-power
+  - id: "LUMI/lumi.switch.n2aeu1"
+    deviceLabel: Aqara Smart Wall Switch H1 (With Neutral, Double Rocker)
+    manufacturer: LUMI
+    model: lumi.switch.n2aeu1
+    deviceProfileName: aqara-switch-power
+  - id: "LUMI/lumi.switch.l1aeu1"
+    deviceLabel: Aqara Smart Wall Switch H1 (No Neutral, Single Rocker)
+    manufacturer: LUMI
+    model: lumi.switch.l1aeu1
+    deviceProfileName: aqara-switch-no-power
+  - id: "LUMI/lumi.switch.l2aeu1"
+    deviceLabel: Aqara Smart Wall Switch H1 (No Neutral, Double Rocker)
+    manufacturer: LUMI
+    model: lumi.switch.l2aeu1
+    deviceProfileName: aqara-switch-no-power    
```

Comment:
you may want to add join name localizations here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/tools/localizations/cn.csv

since the other H1 switches have them

Reply:
Thanks,modified.